### PR TITLE
Release v1.0.4

### PR DESCRIPTION
**Pull request by devonpis**
- update FormioLoader callback
- improve story code snippets to be more meaningful
- improve folder structure

### CODE REVIEW

File: src/examples/SimpleWizard/SimpleWizard.code.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const simpleWizardCode = `
+<head>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio-script.prod.min.js"></script>
+</head>
+<body>
+  <script>
+${indent(
+  SimpleWizard.toString().replaceAll(
+    "_helpers_FormioLoader__WEBPACK_IMPORTED_MODULE_0__",
```
Comment (duttonw) [inline src/examples/SimpleWizard/SimpleWizard.code.js:14]: why do you need to do this?

File: src/examples/SingleSignOn/SingleSignOn.code.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { indent } from "../../utils";
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { SingleSignOn } from "!!./SingleSignOn";
+
+export const singleSignOnCode = `
+<head>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio-script.prod.min.js"></script>
```
Comment (duttonw) [inline src/examples/SingleSignOn/SingleSignOn.code.js:8]: why is there 'prod' in the name?

File: src/examples/SimpleWizard/SimpleWizard.code.js (hunk 1 above)
Comment (devonpis) [reply src/examples/SimpleWizard/SimpleWizard.code.js:14]: this code will show in the code snippet of the story, provides a functional example if the user copy and paste the code to a html page

File: src/examples/SingleSignOn/SingleSignOn.code.js (hunk 2 above)
Comment (devonpis) [reply src/examples/SingleSignOn/SingleSignOn.code.js:8]: formio-script.prod.min.js will load the scripts defined in src/helpers/FormioScript/index.prod.js
formio-script.dev.min.js will load the scripts defined in src/helpers/FormioScript/index.dev.js
etc.

File: src/examples/SimpleWizard/SimpleWizard.code.js (hunk 1 above)
Comment (devonpis) [reply src/examples/SimpleWizard/SimpleWizard.code.js:14]: the function has alredy transpiled by webpack, and webpack has replaced `FormioLoader` with `_helpers_FormioLoader__WEBPACK_IMPORTED_MODULE_0__`, so i have to convert it back...

File: src/helpers/FormioLoader/FormioLoader.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
+import defaultCreateFormOptions from "../../config/createForm.options";
+import defaultCreateFormController from "../../config/createForm.controller";
+
+// polyfill plugin function to fix the namespace option doesn't pass to Formio.makeRequest
+const NamespacePlugin = {
```
Comment (duttonw) [inline src/helpers/FormioLoader/FormioLoader.js:5]: why is this showing up as a new file?

File: src/helpers/FormioScript/FormioScript.stories.mdx [hunk 4]
```diff
@@ -4,9 +4,16 @@ import { Meta } from "@storybook/addon-docs";
 
 # FormioScript
 
-FormioScript provide shorthand implementation that helps loading the necessary scripts that need to load a Formio form in your application.
+FormioScript provide shorthand implementation to lazy load the necessary libraries and scripts that need to initiate a Formio form in your application.
```
Comment (ThrawnCA) [inline src/helpers/FormioScript/FormioScript.stories.mdx:7]: The grammar here could use more work. Perhaps:

"FormioScript provides a shorthand implementation to lazy-load the necessary libraries and scripts to initiate a Formio form in your application."

File: src/examples/SingleSignOn/SingleSignOn.code.js (hunk 2 above)
Comment (devonpis) [reply src/examples/SingleSignOn/SingleSignOn.code.js:8]: maybe the prod one can be renamed to `formio-script.prod.min.js`

File: src/helpers/FormioScript/FormioScript.stories.mdx [hunk 5]
```diff
@@ -4,9 +4,16 @@ import { Meta } from "@storybook/addon-docs";
...
 Simply include it in your application and it will search through the page with Formio div container and initiated the Formio instances.
 
+**Benefits**
+
+- One script and for all, reduce the chance of missing dependency.
+- Enforce the script are loading in certain order, wrong order will break the application, which is prone to error if insert manually.
```
Comment (ThrawnCA) [inline src/helpers/FormioScript/FormioScript.stories.mdx:13]: "script" should be "scripts"

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: src/helpers/FormioLoader/FormioLoader.js (hunk 3 above)
Comment (devonpis) [reply src/helpers/FormioLoader/FormioLoader.js:5]: i've rename src/helpers/FormioLoader/index.js to src/helpers/FormioLoader/FormioLoader.js
seems github didn't detect it as rename action...

**Review by duttonw — APPROVED** [review]

File: src/examples/SingleSignOn/SingleSignOn.code.js (hunk 2 above)
Comment (duttonw) [reply src/examples/SingleSignOn/SingleSignOn.code.js]: if you can, that would be great.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**